### ISSUE TITLE
Reduced mana reserved on Synthesis ring is detected incorrectly as a helmet enchant

### ISSUE THREAD

**Issue by Rohatej** [Kyusung4698/PoE-Overlay#88]
```
Rarity: Unique
Circle of Guilt
Synthesised Iron Ring
--------
Requirements:
Level: 52
--------
Item Level: 82
--------
6% increased Spell Damage per Power Charge (implicit)
--------
+12 to all Attributes
Adds 9 to 15 Physical Damage
+373 to Armour
Herald of Purity has 31% reduced Mana Reservation
48% increased Physical Damage while affected by Herald of Purity
--------
I feel the moral weight of what I must to do to protect Wraeclast.
I accept my guilt without shame. It is my gift to humanity.
--------
Synthesised Item
```
![synth ring](https://user-images.githubusercontent.com/49788153/72351355-7c0ca880-36e0-11ea-86b6-d7f43ece4469.png)

**Comment by Kyusung4698** [comment]
Thanks for reporting. Will be fixed in the next version.

**Comment by vadash** [comment]
Another one

![image](https://user-images.githubusercontent.com/1397582/72397619-7d29ee00-3751-11ea-8b63-cc2009a9371e.png)

**Comment by Kyusung4698** [comment]
Possible fixed with [0.5.0](https://github.com/Kyusung4698/PoE-Overlay/releases/tag/0.5.0). Please let me know if it works! Thanks.

**Comment by Rohatej** [comment]
For Synthesis rings, it's fixed. GJ

![synth fixed](https://user-images.githubusercontent.com/49788153/72476904-ad739a00-37ee-11ea-82b9-a33fa2fa1d70.png)

**Comment by vadash** [comment]
Another one 
```
Rarity: Rare
Brood Scarab
Citrine Amulet
--------
Requirements:
Level: 65
--------
Item Level: 83
--------
+24 to Strength and Dexterity (implicit)
--------
10% reduced Mana Reserved
--------
Shaper Item
Redeemer Item
--------
Note: ~price 60 exa
```

![image](https://user-images.githubusercontent.com/1397582/72490532-7f12b080-3828-11ea-88b2-e18275b7be4e.png)

**Comment by Kyusung4698** [comment]
> Another one
> 
> ```
> Rarity: Rare
> Brood Scarab
> Citrine Amulet
> --------
> Requirements:
> Level: 65
> --------
> Item Level: 83
> --------
> +24 to Strength and Dexterity (implicit)
> --------
> 10% reduced Mana Reserved
> --------
> Shaper Item
> Redeemer Item
> --------
> Note: ~price 60 exa
> ```
> 
> ![image](https://user-images.githubusercontent.com/1397582/72490532-7f12b080-3828-11ea-88b2-e18275b7be4e.png)

Fixed with 0.5.1. Thanks for reporting!

**Comment by vadash** [comment]
Negative values are still fucked

Item
```
Rarity: Rare
Behemoth Shelter
Hubris Circlet
--------
Quality: +28% (augmented)
Energy Shield: 123 (augmented)
--------
Requirements:
Level: 69
Int: 154
--------
Sockets: B-B-G-B 
--------
Item Level: 84
--------
Purity of Lightning has 20% reduced Mana Reservation
--------
+302 to Accuracy Rating
+20 to maximum Energy Shield
+31 to maximum Life
+52 to maximum Mana
13% increased Rarity of Items found
10% reduced Mana Reserved
--------
Redeemer Item
```

![image](https://user-images.githubusercontent.com/1397582/72702696-ab019f00-3b64-11ea-900c-a9b7c750a94e.png)
